scipy/linalg/matfuncs.py

```diff
@@ -668,3 +669,48 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def khatri_rao(a, b):
+    """
+    Khatri-rao product
+
+    Parameters
+    ----------
+    a:  (I, K) ndarray
+        Input array
+    b:  (J, K) ndarray
+        Input array
+
+    Returns
+    -------
+    C:  ((IJ), K) ndarray
+        Khatri-rao product of 'a' and 'b'.
+
+    Notes
+    -----
+    Mathematically:
+
+    A * B = (A_{ij}  \bigotimes B_{ij})_{ij}
+
+    Examples
+    --------
+    >>> from numpy import array
```

Comment:
Please remove this import completely, assume `import numpy as np` is done (don't add that line, it's implicit) and use `np.array` further down.

Reply:
ok